yaml/buddy-error-codes.yaml

```diff
@@ -1346,3 +1346,22 @@ Errors:
     id: "CLOSE_CHAMBER_VENTS"
     type: "CONNECT"
     gui_layout: "warning_dialog"
+
+  - code: "XX839"
+    printers: [COREONE]
+    title: "Chamber Fan Error"
+    text: "Chamber's cooling fan is not spinning. Check it for possible debris, then inspect the wiring."
+    id: "CHAMBER_COOLING_FAN_ERROR"
+    type: "CONNECT"
+    gui_layout: "warning_dialog"
+    approved: true
+
+  - code: "XX840"
+    printers: [XL, COREONE]
+    title: "Chamber Fan Error"
+    text: "Chamber's filtration fan is not spinning. Check it for possible debris, then inspect the wiring."
```

Comment:
Ditto. Also, do we need to specify the "Chamber" here?

Reply:
Open to suggestions. Seems like "Filtration Fan" only could work. Im not sure if every user will get straight away, which one it is.

Reply:
I guess fair point, leaving the "Chamber filtration fan" here is fine 👍